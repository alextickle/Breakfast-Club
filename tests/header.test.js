const Header = require("../bc-react/src/components/Header");
const React = require("react");

it("should render a hardcoded header", () => {
  const wrapper = shallow(<div className="header">Breakfast Club</div>);
  expect(wrapper).toMatchSnapshot();
});
