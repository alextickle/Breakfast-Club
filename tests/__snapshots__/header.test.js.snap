// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a hardcoded header 1`] = `
<div
  className="header"
>
  Breakfast Club
</div>
`;
